coops-ui/src/views/Dashboard.vue

```diff
@@ -244,11 +245,9 @@ export default {
      *
      * @param filings The array of filings in history
      */
-    checkPendingFilings (filings) {
-      filings.forEach(filing => {
-        if (this.entityFilter(EntityTypes.BCOMP) &&
-        filing.name === FilingTypes.ADDRESS_CHANGE &&
-        filing.status === FilingStatus.PAID) {
+    checkPendingFilings (filings: Array<any>) {
+      filings.some(filing => {
```

Comment:
Oops. This was an optimization to stop processing when we found an element, but the function needs to return true/false, which I forgot to add.

Because there is no return value, this will just process every filing (same as before). However I will add a return value in my next PR.